### PR TITLE
Remove sensitivity

### PR DESCRIPTION
**Pull request by haslinghuis**
Level sensitivity was removed in https://github.com/betaflight/betaflight/pull/4013

Fixes: https://github.com/betaflight/betaflight-configurator/pull/644

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Comment by limonspb** [comment]
> Level sensitivity was removed in [betaflight/betaflight#4013](https://github.com/betaflight/betaflight/pull/4013)
> 
> Fixes: #644
> 
> image

nice background :)

**Review by McGiverGim — APPROVED** [review]
I think that for hiding/showing is always better to assign a unique style class to all the elements involved and in this way the hide/show is a lot easier.

**Review by klutvott123 — APPROVED** [review]

**Pull request merged**
